src/plugins/discover/public/plugin.tsx

```diff
@@ -379,11 +384,13 @@ export class DiscoverPlugin
       history: this.historyService.getHistory(),
       urlTracker: this.urlTracker!,
       profilesManager,
-      ebtContextManager,
+      ebtManager,
     });
   };
 
   private registerEmbeddable(core: CoreSetup<DiscoverStartPlugins>, plugins: DiscoverSetupPlugins) {
+    const ebtManager = new DiscoverEBTManager(); // It is not initialized outside of Discover
```

Comment:
We may want to consider also tracking the custom events in embeddables too since we don't have the same issue as we do with context. We can discuss first and do it as a followup though if we decide to.